release/action.yaml

```diff
@@ -40,17 +40,37 @@ runs:
         persist-credentials: false
     - name: Setup tools
       uses: open-turo/action-setup-tools@v1
-    - name: Install dependencies
+    - name: Check for yarn.lock
+      id: check_yarn_lock
+      uses: andstor/file-existence-action@v2
+      with:
+        files: yarn.lock
+    - name: Install dependencies (yarn)
+      if: steps.check_yarn_lock.outputs.files_exists == 'true'
       shell: bash
       run: yarn --pure-lockfile
```

Comment:
Unrelated: I wonder if removing `--pure-lockfile` would allow this to be used with yarn 2+

Reply:
That would not respect the lockfile in yarn v1 projects. Probably we need one more check here to see if the project is using pnp?

Reply:
Or we can drop pnp altogether 😄